### PR TITLE
Define master_count so that baremetal works again. Broken by PR #468.

### PR DESCRIPTION
**Pull request by ggreer**
Fixes https://github.com/coreos-inc/tectonic-issues/issues/240

Regression introduced in #468

### CODE REVIEW

**Comment by s-urbaniak** [comment]
great catch, LGTM!

File: platforms/metal/tectonic.tf [hunk 1]
```diff
@@ -67,6 +67,7 @@ module "tectonic" {
   kubectl_client_id = "tectonic-kubectl"
   ingress_kind      = "HostPort"
   experimental      = "${var.tectonic_experimental}"
+  master_count      = "${var.tectonic_master_count}"
```
Comment (s-urbaniak) [inline platforms/metal/tectonic.tf]: oh, just a second, afaik baremetal does not recognize `tectonic_master_count`, hence what value should we put here?

Comment (s-urbaniak) [inline platforms/metal/tectonic.tf]: I vote for defaulting this to `1` for the metal platform, `tectonic_master_count` is not used there, and in fact https://github.com/coreos/tectonic-installer/pull/536 does not even configure this value anymore for baremetal.

**Comment by ggreer** [comment]
OK it's 1.

File: platforms/metal/tectonic.tf [hunk 2]
```diff
@@ -67,6 +67,7 @@ module "tectonic" {
   kubectl_client_id = "tectonic-kubectl"
   ingress_kind      = "HostPort"
   experimental      = "${var.tectonic_experimental}"
+  master_count      = "1"
```
Comment (Quentin-M) [inline platforms/metal/tectonic.tf]: Should use the length of var.tectonic_metal_controller_names instead.

**Comment by ggreer** [comment]
OK now it's "${len(var.tectonic_metal_controller_names)}"

**Pull request merged**